log/sequencer.go

```diff
@@ -47,6 +47,9 @@ type Sequencer struct {
 	// sequencerGuardWindow is used to ensure entries newer than the guard window will not be
 	// sequenced until they fall outside it. By default there is no guard window.
 	sequencerGuardWindow time.Duration
+	// forceNewRootInterval is used to ensure that a new signed log root is generated after a while,
+	// even if no entries have been added to the log.  Zero duration disables this behavior.
+	forceNewRootInterval time.Duration
```

Comment:
It's still misrepresented and at a larger scope than necessary. LogID is a parameter, so this should be too.